scripts/yarnInstallCi.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+if [ "$1" = "--production" ]; then
+  rm -rf node_modules | yarn workspaces focus --all --production
```

Comment:
ooh what does this do?

Reply:
should we be deleting node_modules?

Reply:
In Yarn v1 when we create a release build in infra using the `--production` parameter, which is passed to `yarn install`, this informs yarn not to install any dev dependencies. 

For some reason this was renamed in v2/v3 to [`yarn workspaces focus --all --production`](https://yarnpkg.com/getting-started/migration#renamed), so I added a the if statement in the script to run that command if the `--production` parameter is passed to the script.

Reply:
@jroebu14 I think someone brought this up previously. The answer is I'm not sure, it was part of the old `yarn ci` script, so I kept it in just in case it might break something, happy to remove it.

Reply:
Thanks for explaining the `yarn workspaces focus --all --production` seems weird the commands are so inconsistent between dev and prod but you've migrated correctly for me 👍

Reply:
Although I can't actually see a use of `./scripts/yarnInstallCi.sh --production`, am I missing something?

Reply:
Yep you're right, it's not used here. It's used in our buildspecs for deployment